### ISSUE TITLE
Design doc for classifier training pipeline from Sagemaker Ground Truth Output

### ISSUE THREAD

**Issue by kschelonka** [AllenInstitute/segmentation-labeling-app#4]
Create design document on confluence to describe how the classifier is trained from Sagemaker Ground Truth outputs.

Include - 
- [ ] MLFlow integration (would likely be very useful here)
- [ ] Feature engineering
- [ ] Train/test/validation set generation
- [ ] Reporting
